src/java/org/apache/cassandra/schema/Tables.java

```diff
@@ -179,6 +181,27 @@ public Tables withUpdatedUserType(UserType udt)
              : this;
     }
 
+    public Tables withNewKeyspace(String newName, Types udts)
+    {
+        Map<String, TableMetadata> updated = new HashMap<>();
+        for (TableMetadata table : getTablesOrderedByDependencies())
+        {
+            updated.put(table.name, table.withNewKeyspace(newName, udts, tables));
+        }
+
+        return builder().add(updated.values()).build();
+    }
+
+    private Set<TableMetadata> getTablesOrderedByDependencies()
```

Comment:
`Set` and `ordered` together? I don't understand

Reply:
Good question. This is leftover from DSE where we traverse graph edges in order. All of this can be deleted, actually since we just need to iterate over `this`.